charts/kured/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.5.1"
 description: A Helm chart for kured
 name: kured
-version: 2.2.2
+version: 2.2.3
```

Comment:
2.2.2 is not released yet, so this is not necessary AFAICS: https://github.com/weaveworks/kured/tree/gh-pages

Reply:
does it matter? the code with a helm chart of 2.2.2 was effectively out in the wild, even though not published through our gh pages...

Which reminds me that we should probably publish the helm chart on merge in master?

Reply:
No, it needs a tag to publish.

cc @ckotzbauer

Reply:
I have a PR to fix this: https://github.com/weaveworks/kured/pull/269

Reply:
I reviewed #269. That's a good change for faster chart releases :+1: